docs/content/providers/rancher.md

```diff
@@ -1,12 +1,14 @@
-# Traefik & Rancher
+# Traefik & Rancher 1.x
```

Comment:
Could you let the title as it was? As Rancher 2.x is mentioned below thanks to your change, the reader should not be "filtered" at the title, but at the note instead.

Reply:
I don't feel strongly either way but I think actually filtering out at the heading level makes some sense; If you are on Rancher 2.x you won't need to read through the detail below to know it's not the right place to be.

Reply:
I agree with @dduportal

Reply:
2 to 1 - resolved. :-)